File: src/c01/1.rb
z = Array.new

z.push(10)

z.push(5+3+4)

z.push(9-1)

z.push(6/2)

z.push((2*4) + (4-6))

a = 3

b = a + 1

z.push(a+b+a*b)

z.push(a==b)

z.push(b>a && (b<(a*b)) ? b : a)

z.push(a == 4 ? 6 : b == 4 ? 6 + 7 + a : 25)

z.push((b > a ? b : a) + 2)

z.push((a > b ? a : a < b ? b : -1) * (a + 1))
 

z.each { |i| puts i }

File: src/c01/1.py
# Hack to print each expression in a separate line
z = []

z.append(10)

z.append(5+3+4)

z.append(9-1)

z.append(6//2)

z.append((2*4) + (4-6))

a = 3

b = a + 1

z.append(a+b+a*b)

z.append(a == b)

z.append(b if b>a and b<a*b else a)

z.append(6 if a==4 else 6+7+a if b==4 else 25)

z.append(2 + (b if b>a else a))

z.append((a if a>b else b if a<b else -1) * (a+1))

print(*z, sep='\n')
